test/unit/smart_answer_flows/adoption_calculator_flow_test.rb

```diff
@@ -104,12 +104,12 @@ class AdoptionCalculatorFlowTest < ActiveSupport::TestCase
             .answer_with(Date.parse("1 November 2017"))
         end
 
-        should "have an earliest leave start date 14 days prior" do
-          assert_equal(Date.parse("1 November 2017"), @question.next_node.a_leave_earliest_start)
+        should "have an earliest start date of 1 November 2017" do
+          assert_equal(Date.parse("1 Novemeber 2017"), @question.next_node.a_leave_earliest_start)
```

Comment:
Typo. Also, today I learnt that Ruby is extremely generous with date parsing.

```
irb(main):017:0> Date.parse("1 NovaScotia 2017")
=> Thu, 01 Nov 2017
```

Reply:
I will fix this before I merge.  Ruby's date parsing is very permissive indeed, let me introduce you to: `Date.parse('23 dogs')`.